src/test/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayoutTest.java

```diff
@@ -0,0 +1,147 @@
+package uk.gov.hmcts.reform.logging.layout;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.joran.spi.JoranException;
+import org.junit.After;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.logging.exception.AbstractLoggingException;
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InvalidClassException;
+import java.io.PrintStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReformLoggingLayoutTest {
+
+    private PrintStream old = System.out;
+    private ByteArrayOutputStream baos = null;
+    private final Logger log = LoggerFactory.getLogger(ReformLoggingLayoutTest.class);
+
+    private enum LogbackConfig {
+        C1("logback-test-enable-thread.xml"),
+        C2("logback-test-custom-date-format.xml"),
+        C3("logback-test-disable-alert-level.xml");
+
+        private String value;
+
+        private LogbackConfig(String configResource) {
+            value = configResource;
+        }
+    }
```

Comment:
Is there a benefit of this enum?
Perhaps just 3 constants would be enough?
Or just inline file names in test (that would be my preference).

Also, C1 C2 C3 sound quite cryptic.